pyaedt/modeler/object3dlayout.py

```diff
@@ -348,6 +348,224 @@ def enabled(self, status=True):
         ]
         return self.change_property(args)
 
+    @property
+    def solderball_enabled(self):
+        """Check if Solderball is enabled.
+
+        Returns
+        -------
+        bool
```

Comment:
```suggestion
        bool
            ````True`` when successful, ``False"" when failed.
```